docs/tutorials/customize_runtime.md

```diff
@@ -157,6 +157,22 @@ We support many other learning rate schedule [here](https://github.com/open-mmla
         min_lr_ratio=1e-5)
     ```
 
+## Customize workflow
+
+Workflow is a list of (phase, epochs) to specify the running order and epochs.
+By default it is set to be
+```python
+workflow = [('train', 1)]
+```
+which means running 1 epoch for training.
+Sometimes user may want to check some metrics (e.g. loss, accuracy) about the model on the validate set.
+In such case, we can set the workflow as
+```python
+[('train', 1), ('val', 1)]
+```
+so that 1 epochs for training and 1 epoch for validation will be ran iteratively.
```

Comment:
1. Do we need to clarify whether the model will be trained on validation set, or the workflow simply performs inference and calculate the loss?
2. In such case, what is the relation ship between this workflow and `total_epochs`, e.g., if `total_epochs=12`, will it train model on training set by 12 epochs, or perform train and val workflows by 6 epochs, respectively?

Reply:
Speaking of that, I think we may also add some description about the equivalence about [('train', 1), ('val", 1)] and [('train', 1) but using evaluation hook.